src/custom/pages/Swap/swapFlow/useSwapFlowContext.ts

```diff
@@ -135,6 +131,14 @@ export function useSwapFlowContext(): SwapFlowContext | null {
   const isSellEth = ETHER.onChain(chainId).equals(v2Trade.inputAmount.currency)
   const isGnosisSafeWallet = !!gnosisSafeInfo
 
+  const sellToken = v2Trade.inputAmount.currency.wrapped
```

Comment:
nit: can we just call this `trade`? we don't use `v2Trade` this was/is a uni artifact

Reply:
Done: https://github.com/cowprotocol/cowswap/pull/1049/commits/c17b9decc6570e42cdea00f039ba335d9116c23c